docs/pages/web/slimbanner.js

```diff
@@ -589,7 +611,11 @@ Tooltip provides helpful information regarding an interactive UI element, typica
 }
 
 export async function getStaticProps(): Promise<{| props: {| generatedDocGen: DocGen |} |}> {
+  const docGen = await docgen({ componentName: 'SlimBanner' });
+
+  docGen.props.primaryAction.flowType.raw = '{| string | React.Element<typeof Text>>> |}';
```

Comment:
Ah, I see now. Let's make this change in GeneratedPropTable, as we want this to apply everywhere we use React.Element

Reply:
FIXED

docGen.props.**message**. flowType.raw